std_msgs/Header header
time time_send
uint32 frame_number
uint8 camera_id
grsim_ros_bridge/Ball[] Balls
grsim_ros_bridge/Robot[] Robots
